modules/http/src/main/java/org/mule/extension/http/api/policy/HttpRequestPolicyPointcutParameters.java

```diff
@@ -46,4 +50,11 @@ public String getPath() {
   public String getMethod() {
     return method;
   }
+
+  /**
+   * @return the name of the flow executing the http request
+   */
+  public String getFlowName() {
+    return flowName;
```

Comment:
What is the flow name used for? Doe sit make sense to have it in PolicyPointcutParameters instead?